iree_tests/pytorch/models/sd-vae-decode-tank/test_cases.json

```diff
@@ -0,0 +1,35 @@
+{
+  "test_cases": [
+    {
+      "name": "splats",
+      "runtime_flagfile": "splat_data_flags.txt",
+      "remote_file_groups": [
+        {
+          "azure_account_url": "https://sharkpublic.blob.core.windows.net",
+          "azure_container_name": "sharkpublic",
+          "azure_base_blob_name": "sai/sd-vae-decode-tank/",
+          "files": [
+            "sd-vae-decode-tank.mlirbc"
+          ]
```

Comment:
FYI, I'm considering collapsing these back down to just URLs like
`https://sharkpublic.blob.core.windows.net/sharkpublic/sai/sd-vae-decode-tank/sd-vae-decode-tank.mlirbc`
I was trying to be fancy with Azure downloading and saving a few characters in .json files but making the URLs easy to copy/paste and `wget` is going to be nicer for developers

Reply:
Ok cool. This is also pretty nice though. Just being able to see the file names instead of the cluttered urls can be nice for anyone taking a look. Maybe we can add a comment that the base url to each file like: `https://sharkpublic.blob.core.windows.net/sharkpublic/sai/sd-vae-decode-tank` is the base url. If interested in downloading the file simply wget base url + `/{file_name}`

Reply:
Sure. I at least consider what's in the json files today to be very rough V0 code. So I wouldn't treat it as sacred if you find it tricky to work with.

Another ideas was to group files for each test case using tar/zip, so we wouldn't need to list them all explicitly.

Reply:
Oooh that would be much easier for adding the tests